_data/gaming.yml

```
@@ -140,7 +140,7 @@ websites:
     img: gog.png
     tfa:
       - email
-    doc: https://www.gog.com/support/support_view/twostep_login_faq
+    doc: https://support.gog.com/hc/en-us/articles/115003660533-What-is-two-step-login-and-how-does-it-work-
```

```suggestion
    doc: https://support.gog.com/hc/en-us/articles/115003660533
```
For HelpCenter URLs we do not need the slug after the ID.